src/go/k8s/controllers/redpanda/console_controller_test.go

```diff
@@ -486,4 +486,32 @@ var _ = Describe("Console controller", func() {
 			Expect(envs[0].ValueFrom.SecretKeyRef.Name).Should(Equal(secretName))
 		})
 	})
+
+	Context("When ConfigMap is deleted", func() {
+		ctx := context.Background()
+		It("Should reconcile and recreate the ConfigMap", func() {
+			By("Getting Console")
+			consoleLookupKey := types.NamespacedName{Name: ConsoleName, Namespace: ConsoleNamespace}
+			createdConsole := &redpandav1alpha1.Console{}
+			Expect(k8sClient.Get(ctx, consoleLookupKey, createdConsole)).Should(Succeed())
```

Comment:
How console was created? I don't see creation in 
https://github.com/redpanda-data/redpanda/blob/e7c235c15c0bfd0eaa7846431cec919d7c8074d2/src/go/k8s/controllers/redpanda/suite_test.go#L163-L176

I see this test relies on another test being run before https://github.com/redpanda-data/redpanda/blob/e7c235c15c0bfd0eaa7846431cec919d7c8074d2/src/go/k8s/controllers/redpanda/console_controller_test.go#L58-L64.

Each test case should be dependent of each other.